src/features/pilot_management/PilotSheet/components/LevelEditDialog.vue

```diff
@@ -20,35 +20,33 @@
         <v-col cols="auto">
           <span class="overline">Current Level:</span>
           <br />
-          <v-text-field
-            v-model="pilot.Level"
-            readonly
-            type="number"
-            hide-details
-            dense
-            outlined
-            background-color="panel"
+          <span
+            style="display:inline-block"
             class="level-input"
-          />
+          >
+            {{ pilot.Level }}
+          </span>
         </v-col>
 
         <v-col cols="auto" class="mx-3">
+          <span class="overline"></span>
+          <br />
           <v-icon x-large>arrow_forward</v-icon>
         </v-col>
 
         <v-col cols="auto">
           <span class="overline">New Level:</span>
           <br />
-          <v-text-field
+          <v-select
             v-model="newLevel"
+            :items="levels"
             type="number"
-            maxlength="2"
             hide-details
+            hide-spin-buttons
```

Comment:
This was a built-in option for hiding Spin Buttons; it seemed reasonable to use a builtin instead of a custom style.